stat/distuv/alphastable.go

```diff
@@ -17,11 +17,15 @@ import (
 //  location μ
 // More information at https://en.wikipedia.org/wiki/Stable_distribution
 type AlphaStable struct {
+	// Stability parameter 0 < α ≤ 2.
```

Comment:
These repeat what exists in the comment for the type.

```
// AlphaStable represents an α-stable distribution with four parameters.
// See https://en.wikipedia.org/wiki/Stable_distribution for details.
type AlphaStable struct {
	// Alpha is the stability parameter for the distribution.
	// it is valid within the range 0 < α ≤ 2.
	Alpha float64
	// Beta is the skewness parameter for the distribution.
	// it is valid within the range -1 ≤ β ≤ 1.
	Beta float64
	// C is the scale parameter for the distribution.
	// it is valid when positive.
	C float64
	// Mu is the location parameter for the distribution.
	Mu  float64

	Src rand.Source
}
```

Reply:
Good point. Updated the comment. PTAL.